### DISCUSSION TITLE
Trying to validate uploads with Livewire

### DISCUSSION THREAD

**Discussion by elfeffe** [spatie/laravel-medialibrary#2706] [Q&A]
Hello,
I'm trying to validate using a Livewire component (I'm using Medialibrary Pro).

This is my validation
```

    public $mediaComponentNames = ['uploads'];
    public $uploads;

protected function rules()
    {
        $rules = [
            'deliveryNote.reference' => 'string|nullable',
            'deliveryNote.can_split' => 'boolean|nullable',
            'deliveryNote.no_label' => 'boolean|nullable',
            'address.id' => 'int',
            'carrierSelect' => 'nullable',
            'items.*.id' => 'int',
            'items.*.qty' => 'int',
        ];

        if ($this->deliveryNote->no_label) {
            $rules['uploads'] = $this
                ->validateMultipleMedia()
                ->minItems(1)
                ;
        };

        return $rules;
    }


```
I get this message
`Spatie\MediaLibraryPro\Rules\UploadedMediaRules
`

And I get no validation.
Any help?

**Comment by euoia** [comment]
Hi @elfeffe - I have a similar issue - did you resolve your problem?

**Reply by elfeffe** [reply]
No. I’m not validating them.

**Reply by euoia** [reply]
I found that this issue went away when I added the `use ValidatesMedia` trait to a class extending `FormRequest`. I had been trying to use the `ValidatesMedia` trait directly on the controller, but this doesn't seem to work for whatever reason. By creating a separate `FormRequest`, I am able to validate the image.

**Reply by euoia** [reply]
@elfeffe thanks for your quick response to my question. I've created an issue because I would like a bit more information about this: https://github.com/spatie/laravel-medialibrary-pro/issues/375
